README.md

```diff
@@ -74,16 +79,39 @@ with **at least** the following roles:
   - Get and list access to GKE Clusters.
 `
 
-### Via Credentials
+### Via google-github-actions/auth
+
+Use [google-github-actions/auth](https://github.com/google-github-actions/auth) to authenticate the action. You can use [Workload Identity Federation][wif] or traditional [Service Account Key JSON][sa] authentication.
+by specifying the `credentials` input. This Action supports both the recommended [Workload Identity Federation][wif]
```

Comment:
I think the git diff got weird here because I did a web-based suggestion. I think you want to delete L85-86

Reply:
should be fixed now